backend/FwLite/LcmCrdt.Tests/Changes/ChangeSerializationTests.cs

```diff
@@ -1,26 +1,36 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Text.Json;
 using FluentAssertions.Execution;
 using LcmCrdt.Changes;
 using LcmCrdt.Changes.Entries;
 using MiniLcm.Tests.AutoFakerHelpers;
 using SIL.Harmony.Changes;
+using SIL.WritingSystems;
 using Soenneker.Utils.AutoBogus;
 using SystemTextJsonPatch;
 
 namespace LcmCrdt.Tests.Changes;
 
 public class ChangeSerializationTests
 {
+    private static readonly Lazy<JsonSerializerOptions> LazyOptions = new(() =>
+    {
+        var config = new CrdtConfig();
+        LcmCrdtKernel.ConfigureCrdt(config);
+        config.JsonSerializerOptions.ReadCommentHandling = JsonCommentHandling.Skip;
+        return config.JsonSerializerOptions;
+    });
+    private static readonly JsonSerializerOptions Options = LazyOptions.Value;
```

Comment:
You access the Lazy value immediately. So, I guess you're just doing it so you can do the setup in a lambda?

Reply:
Yeah pretty much